src/onnxruntime.cc

```diff
@@ -1005,6 +1005,13 @@ ModelInstanceState::ModelInstanceState(
       cuda_allocator_info_(nullptr), cpu_allocator_info_(nullptr),
       io_binding_(nullptr), output_buffer_(nullptr)
 {
+  // Segmentation fault on server destruction is observed under parallel model
+  // loading when running L0_onnx_optimization if this construction is not
+  // serialized.
+  // [FIXME] TITLE (DLIS-XXXX)
```

Comment:
Don't forget to add this information (needed before approval).

Reply:
Yes, updated along with the commit: [Reduce lock scope to ort create session](https://github.com/triton-inference-server/onnxruntime_backend/pull/174/commits/2899f7c90f817a41470967f2fe41aa1fc9fc4d56)